analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -86,13 +86,19 @@ class SpdxDocumentFile(
     /**
      * Return the dependencies of [pkg] defined in [doc] of the [SpdxRelationship.Type.DEPENDENCY_OF] type.
      */
-    private fun getDependencies(pkg: SpdxPackage, doc: SpdxDocument): SortedSet<PackageReference> =
-        getDependencies(pkg, doc, SpdxRelationship.Type.DEPENDENCY_OF) { target ->
+    private fun getDependencies(
```

Comment:
The KDoc comment for the function should list the new parameters, too. The same applies to some other functions below.